_redirects

```diff
@@ -12,3 +12,7 @@
 /*/contribute https://wiki.almalinux.org/Contribute.html 302
 /*/contribute/ https://wiki.almalinux.org/Contribute.html 302
 
+
+# added for marketing reasons
+/fips/ /blog/2023-09-19-fips-validation-for-almalinux/ 301
```

Comment:
We shouldn't make this a 301/perm but a 302 as it's conceivable we will have a /fips landing page in the future.

Reply:
good call! fixed.